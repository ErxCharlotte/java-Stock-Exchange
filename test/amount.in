ADD trader1 100.55
ADD trader2 20.00
IMPORT trader1 ABC 100
IMPORT trader1 CDE 1000
EXPORT trader1 CDE 1000
Amount trader1 CDE
AMOUNT trader1 ABC
AMOUNT trader1 LOL
AMOUNT trader2 ABC
AMOUNT trader3 ABC
EXIT