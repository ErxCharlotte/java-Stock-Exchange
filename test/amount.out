$ Success.
$ Success.
$ Trader now has 100.00 units of ABC.
$ Trader now has 1000.00 units of CDE.
$ Trader now has no units of CDE.
$ Product not in inventory.
$ 100.00
$ Product not in inventory.
$ Product not in inventory.
$ No such trader in the market.
$ Have a nice day.
